docs/source/api/v13/federation.rst

```diff
@@ -0,0 +1,449 @@
+.. 
```

Comment:
Since this PR rewrites an existing endpoint with the same functionality, the existing API doc should be updated rather than copying the doc into 1.3 and updating.

I believe 1.3 APIs are for _new_ endpoints that only exist in traffic_ops_golang